package-lock.json

```diff
@@ -28,11 +28,1665 @@
       },
       "devDependencies": {
         "autoprefixer": "^10.4.13",
+        "next-pwa": "^5.6.0",
         "postcss": "^8.4.20",
         "serve": "^14.2.0",
         "tailwindcss": "^3.2.4"
       }
     },
+    "node_modules/@ampproject/remapping": {
```

Comment:
Damn, this thing adds a lot of dependencies. I saw it's about 50mb extra. So be it!

Reply:
I know, but `next-pwa` seems to be the canonical way of building PWA nextJS applications. 
In any case the performance benefits are bigger, caching static resources etc